gslab_scons/tests/test_size_warning.py

```diff
@@ -88,24 +121,36 @@ def test_list_ignored_files(self, mock_check, mock_walk,
         mock_isdir.side_effect  = isdir_ignored_side_effect
         mock_isfile.side_effect = isfile_ignored_side_effect
 
+        # Multiple directories
         look_in = ['raw', 'release']
         ignored = sw.list_ignored_files(look_in)
         expect_ignored = ['raw/large_file.txt',
                           'release/.DS_Store', 
                           'release/subdir/ignored.txt']
-        print ignored
+
         self.assertEqual(len(ignored), len(expect_ignored))
         for i in range(len(ignored)):
             self.assertIn(ignored[i], expect_ignored)
 
-
+        # One directory
         look_in = ['raw']
         ignored = sw.list_ignored_files(look_in)
         expect_ignored = ['raw/large_file.txt']
 
         self.assertEqual(len(ignored), len(expect_ignored))
         self.assertEqual(ignored[0], expect_ignored[0])       
 
+        # The root
+        look_in = ['.']
+        ignored = sw.list_ignored_files(look_in)
+        expect_ignored = ['root_ignored.txt',
+                          'raw/large_file.txt',
+                          'release/.DS_Store',
+                          'release/subdir/ignored.txt']
+
+        self.assertEqual(len(ignored), len(expect_ignored))
+        self.assertEqual(ignored[0], expect_ignored[0])          
```

Comment:
I like the `for i in range(len(ignored)):` check that we do in lines 132-133 a bit better.